bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -423,7 +436,7 @@ private void transformAndStreamFile(SlingHttpServletResponse response, ValueMap
                     || "svg".equalsIgnoreCase(mimeTypeService.getExtension(imageType))) {
                 LOGGER.debug("GIF or SVG file detected; will render the original file.");
                 if (is != null) {
-                    stream(response, is, imageType);
+                    stream(response, is, imageType,imageName);
```

Comment:
Space missing:
`stream(response, is, imageType, imageName);`

Reply:
fixed